BatteryFactor.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BatteryManagement
+{
+    public class BatteryFactor

```

Comment:
1.Class Name and Function Argument is same

Reply:
Updated the file